buzzmobile/sense/lidar/lidar.py

```diff
@@ -1,24 +1,55 @@
 import cv2
 import numpy as np
+import constants
```

Comment:
Does this import work given that `constants.py` is in `include/`?
Does ROS just find these files and links them? And/or did you test this in ROS?